### PR TITLE
Updating from V2 to V3 Schema

### PR DESCRIPTION
**Pull request by a27cheung**
Changes include:

    # ${base_url}/v3/aggregated.csv.gz
    # .../v3/epidemiology.csv
    # .../v3/demographics.csv
    # .../v3/latest/aggregated.csv.gz <- Not ready yet!!!
    # .../v3/latest/epidemiology.csv <- 1 row per location_key

    # .../v2/${location_key}/main.csv
    # .../v3/location/${location_key}.csv
    # go/open-covid-data -> GCS Bucket where the data lives
    #"df = pd.read_csv(f'https://storage.googleapis.com/covid19-open-data/v2/{data_key}/main.csv').set_index('date')"
    "df = pd.read_csv(f'https://storage.googleapis.com/covid19-open-data/v3/location/{data_key}.csv').set_index('date')"

### CODE REVIEW

File: examples/exponential_modeling.ipynb [hunk 1]
```diff
@@ -37,7 +37,17 @@
...
+    # ${base_url}/v3/aggregated.csv.gz
+    # .../v3/epidemiology.csv
+    # .../v3/demographics.csv
+    # .../v3/latest/aggregated.csv.gz <- Not ready yet!!!
+    # .../v3/latest/epidemiology.csv <- 1 row per location_key
+
+    # .../v2/${location_key}/main.csv
+    # .../v3/location/${location_key}.csv
+    # go/open-covid-data -> GCS Bucket where the data lives
+    #"df = pd.read_csv(f'https://storage.googleapis.com/covid19-open-data/v2/{data_key}/main.csv').set_index('date')"
```
Comment (owahltinez) [inline examples/exponential_modeling.ipynb:49]: This is not valid formatting of ipynb files, how did you modify these?

**Review by owahltinez — CHANGES_REQUESTED** [review]

File: examples/exponential_modeling.ipynb (hunk 1 above)
Comment (a27cheung) [reply examples/exponential_modeling.ipynb:49]: ...by writing software like a hardware engineer. Fail. :( Thanks for the feedback! Re-factoring. :D

**Review by a27cheung — COMMENTED** [review]
Deleted all the notes in comments.

**Review by owahltinez — APPROVED** [review]
LGTM, thanks!

**Pull request merged**